x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -319,3 +322,25 @@ export interface AlertingIndexUIQueryParams {
   date_range?: string;
   filters?: string;
 }
+
+export interface GetDatasourcesResponse {
+  items: PolicyData[];
+  total: number;
+  page: number;
+  perPage: number;
+  success: boolean;
+}
+
+export interface GetDatasourceResponse {
+  item: PolicyData;
+  success: boolean;
+}
+
+export type UpdateDatasourceResponse = CreateDatasourceResponse & {
+  item: PolicyData;
+};
+
+/**
+ * The PageId type is used for the payload when firing userNavigatedToPage actions
+ */
+export type PageId = 'alertsPage' | 'managementPage' | 'policyListPage';
```

Comment:
FYI - with the PR I'm working on now, I should be able to delete this along with the hook that uses it. :)